### PR TITLE
Set up local testing server for integration with dashboard

### PR DESCRIPTION
**Pull request by jmkulwik**
This sets up a minimal local websocket server so developers can work on code studio/dashboard at the same time as javabuilder.

Follow-up work on dashboard in order to use this change is here: https://github.com/code-dot-org/code-dot-org/pull/40116

### CODE REVIEW

File: org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+   */
+  @OnOpen
+  public void onOpen(Session session) {
+    // Temporary project url until the dashboard side is integrated
+    String projectUrl = "http://localhost-studio.code.org:3000/v3/files/MoGwvaZmQSQZaImS3LliCQ";
+    String[] fileNames = new String[] {"MyClass.java"};
+
+    outputAdapter = new WebSocketOutputAdapter(session);
+    inputAdapter = new WebSocketInputAdapter();
+    final UserProjectFileManager fileManager = new UserProjectFileManager(projectUrl, fileNames);
```
Comment (molly-moen) [inline org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java:40]: my recently merged PR changed the constructor for UserProjectFileManager, so you'll need to update this to no longer include filenames

**Review by molly-moen — COMMENTED** [review]
Clarifying question: we have some follow-up work to do on dashboard to make this work for local development, correct?

**Comment by jmkulwik** [comment]
> Clarifying question: we have some follow-up work to do on dashboard to make this work for local development, correct?

Yes. I'll go ahead and link the draft PR where I'm working on the dashboard change.

File: org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java (hunk 1 above)
Comment (jmkulwik) [reply org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java]: Fixed. Thanks!

**Review by molly-moen — APPROVED** [review]
Looks good to me!

**Pull request merged**